hmt_escrow/job.py

```diff
@@ -212,7 +212,8 @@ def __init__(self,
                  credentials: Dict[str, str],
                  escrow_manifest: Manifest = None,
                  factory_addr: str = None,
-                 escrow_addr: str = None):
+                 escrow_addr: str = None,
+                 multi_credentials: list = []):
```

Comment:
Might be good to add a `list[Tuple]` to the type here

Reply:
Fixed